k8s-daemonset/k8s-ingress/nginx/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM nginx
```

Comment:
Per comment above, I'd moved all of the files needed to build to docker images into a `docker/nginx-headers-more` dir or something similar.